tests/client/src/components/Carousel.test.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
-import Enzyme, { shallow } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
-import Carousel from '../../../../client/src/components/Carousel';
+import { shallow } from 'enzyme';
+import Carousel from '../../../../client/src/components/carousel';
```

Comment:
I think the benefit of Jest is that you don't need a tests folder and can put your test right next to your component. That way you wouldn't have to do "../../../../"

Reply:
Yeah, currently having it like this because the test folder was requested, so thought it would be easier to bundle them in one area.